src/odemis/acq/stream/_projection.py

```diff
@@ -347,7 +357,7 @@ def _projectTile(self, tile):
             return tile
```

Comment:
If it's 3D, it should have the metadata MD_POS updated, so that it represents the current stack.